### PR TITLE
Weechat

### PR DESCRIPTION
**Pull request by AlanSherman**
stuff here

### CODE REVIEW

File: content/guides/weechat,advanced.mkd [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+buffers. Once installed you can toggle the bar on and off using `/bar toggle
+buffers`. If that's not fast enough for you we can make a new keybind `/key
+bind meta-b /bar toggle buffers` now simply hit Alt-b to toggle the buffer
+list. You might also notice that the buffers sidebar has it own color scheme. If
+you pull up iset and search for 'buffers' you will see that its color
+options are very similar to nd meta-b /bar toggle buffersthe status bar options we were working with before.
+
+### beep
+
+The next recommended pugin is the beep plugin this plugin will let you perform
```
Comment (justinnoah) [inline content/guides/weechat,advanced.mkd]: pugin

File: content/guides/weechat,advanced.mkd [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+of any nicks/chans you do or don't want to be notify on. keep in mind at this
+point we are passing weechat though many layers, screen, remote shell, ssh,
+local shell, terminal, so actually getting alerts on your local system
+can be tricky. For the time being diving into beep is outside the scope of this
+guide. there are also many alternatives for this script, some of which are much
+more powerful, search for notify to see more then a few.
+
+### urlserver
+
+The finale recommended plugin is urlserver. when running in a smaller window
```
Comment (justinnoah) [inline content/guides/weechat,advanced.mkd]: final

**Comment by lucywyman** [comment]
Is this still relevant, or can I go ahead and close it?  @AlanSherman @ElijahCaine  @edunham

**Comment by edunham** [comment]
@lucywyman I think it's valuable to have the guide. You can add Alan's repo as a remote, cherry-pick the commits onto a local branch, then do whatever cleanup you want before making a new PR.

**Comment by lucywyman** [comment]
Closed in favor of #71

**Comment by lucywyman** [comment]
Closed in favor of #71

**Pull request closed without merging**